library/core-utils/src/main/java/im/vector/lib/core/utils/flow/TimingOperators.kt

```diff
@@ -68,10 +69,10 @@ fun <T> Flow<T>.chunk(durationInMillis: Long): Flow<List<T>> {
 
 @ExperimentalCoroutinesApi
 fun <T> Flow<T>.throttleFirst(windowDuration: Long): Flow<T> = flow {
-    var windowStartTime = System.currentTimeMillis()
+    var windowStartTime = SystemClock.elapsedRealtime()
```

Comment:
ideally we'd make use of the `Clock` here as directly interacting with system isn't too test friendly (and have the `Clock` as part of `core`), the but I understand this is a quick workaround to get the CI green again

Reply:
I agree, yes, quick fix for now.
Also will be a bit painful to pass a `Clock` to every `throttleFirst` call.